### DISCUSSION TITLE
Add fallback state to image

### DISCUSSION THREAD

**Discussion by yceballost** [Telefonica/mistica-design#728] [Proposals]
Cómo deberíamos gestionar el estado de fallback de una imagen?

Debería aparecer un placeholder? Debería usar un skeleton indefinidamente?

**Comment by MontseTelefonica** [comment]
Opiniones:
-Un skeleton indefinido puede generar la sensación de que va a cargar en algún momento y si esperas lo verás... y no :P
-la duda es: ¿debe el placeholder servir para informar al usuario que ha habido un error de carga en la imagen o simplemente "rellenar" ese espacio ?

**Reply by yceballost** [reply]
1. Estoy de acuerdo en no usar un skeleton indefinidamente, el skeleton debería mostrarte un estado de carga real y no algo que falló. (solo lo puse porque en alguna conversación salió esa alternativa y por tenerla recogida)
2. Creo que ambas pueden convivir. 
	
Es decir, si no carga la imagen de una Media Card creo que no serviría meter una imagen que rellene porque en ese caso, si usaste una media card creo que es porque la imagen realmente le aportará mucho a un usuario y aquí igual si que mostraría que la imagen no se ha cargado correctamente. La venta de un telefóno por ejemplo.

Sin embargo, en una row si no carga un icono no me parece importante que el usuario sepa que ese icono no se ha cargado porque sinceramente, un icono en pocos casos aporta valor (unicamente en aquellos casos donde está muy estandarizado, en los casos de listas con iconos... meh)
